library/psa_crypto.c

```diff
@@ -6481,6 +6481,47 @@ static psa_status_t psa_key_derivation_input_internal(
     return status;
 }
 
+static psa_status_t psa_key_derivation_input_integer_internal(
+    psa_key_derivation_operation_t *operation,
+    psa_key_derivation_step_t step,
+    uint64_t value)
+{
+    psa_status_t status;
+    psa_algorithm_t kdf_alg = psa_key_derivation_get_kdf_alg(operation);
+
+#if defined(BUILTIN_ALG_ANY_HKDF)
```

Comment:
This is correct but I'm not sure if it's the more efficient way for doing it, in terms of code size and maintainability. Instead of list all the algorithms that don't allow this, it seems simpler to me to list the one alg that does allow this. Or am I missing something?